packages/core/src/contrib/Constraints.ts

```diff
@@ -564,7 +529,8 @@ export const constrDict = {
     padding = 10
   ) => {
     if (isRectlike(t1) && t2 === "Circle") {
-      const textR = max(s1.w.contents, s1.h.contents);
+      const s1BBox = bbox([t1, s1]);
```

Comment:
Avoid using bbox if not necessary

Reply:
This is necessary b/c Square does not have `w` or `h`.